app/mailers/distribution_mailer.rb

```diff
@@ -14,4 +14,10 @@ def partner_mailer(current_organization, distribution, subject)
     attachments[format("%s %s.pdf", @partner.name, @distribution.created_at.strftime("%Y-%m-%d"))] = DistributionPdf.new(current_organization, @distribution).render
     mail(to: @partner.email, from: @from_email, subject: subject)
   end
+
+  def reminder_email(distribution)
+    @partner = distribution.partner
+    @distribution = distribution
+    mail(to: @partner.email, from: "me@diaper.com", subject: "Diaper Distribution Reminder")
```

Comment:
Does the `@from` field get overwritten later? 

If not, we should probably use an actual email

Reply:
My bad here. I added that email "for placement only." Wasn't sure which email to use and I forgot to ask.